sirepo/template/cloudmc.py

```diff
@@ -279,3 +309,160 @@ def _generate_space(space):
     else:
         raise AssertionError("unknown space type: {}".format(space._type))
     return _generate_call(space._type, args)
+
+
+def _generate_tallies(data):
+    if not len(data.models.settings.tallies):
+        raise AssertionError(f"No Tallies defined")
+    return (
+        "\n".join(
+            [
+                _generate_tally(t, data.models.volumes)
+                for t in data.models.settings.tallies
+            ]
+        )
+        + f"""
+tallies = openmc.Tallies([
+    {','.join(['t' + str(tally._index + 1) for tally in data.models.settings.tallies])}
+])
+tallies.export_to_xml()
+"""
```

Comment:
No problem, but I'm curious when to use this construction instead of just putting everything in the jinja file